doc/transform_landmarked_pumap.rst

```diff
@@ -0,0 +1,227 @@
+
+To make samples from ``N`` Stand out more, we'll over-plot them in black.
+
+.. code:: python3
+
+    emb_x2 = embedder.transform(x2)
+
+.. code:: python3
+
+    plt.scatter(emb_x2[:,0], emb_x2[:,1], c=y2, cmap='Spectral', s=2, alpha=0.2)
+    plt.scatter(emb_x2[y2==N][:,0], emb_x2[y2==N][:,1], c='k', s=2, alpha=0.5)
+
+.. image:: images/retrain_pumap_emb_x2.png
+
+While our ``UMAP`` embedder has correctly handled the classes present in ``x1`` it has treated examples from our holdout class ``N`` poorly. Many of these points are concentrated on top of existing classes, with some spread out between them. This inability to generalize is not unique to UMAP, but is more generally a difficulty with learned embeddings. It also may or may not be an issue, depending on your use case. 
+
+New data with Parametric UMAP
+-----------------------------
+
+We can improve this outcome with Parametric UMAP. Parametric UMAP differs from UMAP in that it learns the relationship between the data and embedding with a neural network, instead of learning embeddings directly. This means we can incorporate new data by continuing to train the neural network, updating the weights to incorporate our new information. 
+
+.. image:: images/pumap-only.png
+
+For more complete information on Parametric UMAP and the many options it provides, see :doc:`parametric_umap`.  
+
+We will start adressing this by training a ``ParametricUMAP`` embedding model, and running the same experiment:
+
+.. code:: python3
+
+    p_embedder = ParametricUMAP()
+    
+    p_emb_x1 = p_embedder.fit_transform(x1)
+
+.. code:: python3
+
+    plt.scatter(p_emb_x1[:,0], p_emb_x1[:,1], c=y1, cmap='Spectral', s=2, alpha=0.2)
+
+.. image:: images/retrain_pumap_p_emb_x1.png
+
+Again, we get good results on our initial embedding of ``x1``. If we pass ``x2`` through without re-training, we get a similar problem to our ``UMAP`` model:
+
+.. code:: python3
+
+    p_emb_x2 = p_embedder.transform(x2)
+
+.. code:: python3
+
+    plt.scatter(p_emb_x2[:,0], p_emb_x2[:,1], c=y2, cmap='Spectral', s=2, alpha=0.2)
+    plt.scatter(p_emb_x2[y2==N][:,0], p_emb_x2[y2==N][:,1], c='k', s=2, alpha=0.5)
+
+.. image:: images/retrain_pumap_p_emb_x2.png
+
+Re-training Parametric UMAP with landmarks
+------------------------------------------
+
+To update our embedding to include the new class, we'll fine-tune our existing ``ParametricUMAP`` model. Doing this without any other changes will start from where we left off, but our embedding space's structure may drift and change. This is because the UMAP loss function is invariant to scaling, translation, and rotation, as it is only concerned with the relative positions and distances between points. 
```

Comment:
The loss is not actually completely invariant to scaling.

Reply:
I'm actually glad to hear this, I had a niggling worry that I'd have to scale the landmark loss based on the extents of the existing embedding. I'll update this line.